Change dependabot update strategy

moiikana opened the pull request:
- make dependabot group minor updates
- make dependabot look for patch updates only monthly

https://phabricator.wikimedia.org/T393571

Abban reviewed (approved):
Looks good to me. The old banners are creeping back though 😭

moiikana committed:
Change dependabot update strategy

- make dependabot group minor updates
- make dependabot look for patch updates only monthly

https://phabricator.wikimedia.org/T393571
